.stylelintrc

```diff
@@ -1,5 +1,6 @@
 { 
   "extends": "stylelint-config-recommended",
```

Comment:
Can you please add the corresponding rules to the stylelint config file for oppia.css and make the corresponding fixes as well? Once that has been done, this PR will be good to go (and all tests pass)?